samples/snippets/materialized_view.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def create_materialized_view(override_values={}):
+    # [START bigquery_create_materialized_view]
+    from google.cloud import bigquery
+
+    bigquery_client = bigquery.Client()
+
+    view_id = "my-project.my_dataset.my_materialized_view"
+    base_table_id = "my-project.my_dataset.my_base_table"
+    # [END bigquery_create_materialized_view]
+    # To facilitate testing, we replace values with alternatives
+    # provided by the testing harness.
+    view_id = override_values.get("view_id", view_id)
+    base_table_id = override_values.get("base_table_id", view_id)
+    # [START bigquery_create_materialized_view]
+    view = bigquery.Table(view_id)
+    view.mv_query = f"""
+    SELECT product_id, SUM(clicks) AS sum_clicks
+    FROM  `{base_table_id}`
+    GROUP BY 1
+    """
+
+    # Make an API request to create the materialized view.
+    view = bigquery_client.create_table(view)
+    print(f"Created {view.table_type}: {str(view.reference)}")
+    # [END bigquery_create_materialized_view]
+    return view
+
+
+def alter_materialized_view(override_values={}):
+    # [START bigquery_alter_materialized_view]
```

Comment:
This tag makes me itchy, and expecting a DDL style ALTER MATERIALIZED VIEW statement rather than an api update call.  Should we correct the tag?

Reply:
Agreed, corrected to `bigquery_update_materialized_view`.

The corresponding docs section has room for both. https://cloud.google.com/bigquery/docs/materialized-views#alter For example the `bq` table shows `bq query` with DDL and `bq update` options.